cypress/e2e/dfi.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { exchanges } from "../fixture/dfi.config";
```

Comment:
Let's create a base folder "explore" then split it per pages. Instead of putting it on the root.

e2e/explore:
1. "dfi"
2. "dex"
3. "wallets"

and all sub routes of Explore